newsroom/templates/download_item.txt

```diff
@@ -1,11 +1,7 @@
 {{ item.slugline }}
 {{ item.headline }}
+{{ item.source }}
```

Comment:
This changes the default format of text downloads. Might need this modification in the customer's theme folder instead